core/json/json_parser.cc

```diff
@@ -18,21 +18,38 @@ namespace json {
 std::optional<JsonParser> JsonParser::Parser(const std::string& path) noexcept {
   std::ifstream f(path);
   if (!f.is_open()) {
-    return {};
+    return std::nullopt;
+  }
+  std::stringstream buf;
+  buf << f.rdbuf();
+  f.close();
+  std::string content = buf.str();
+  if (!nlohmann::json::accept(content, true)) {
+    return std::nullopt;
   }
-  return std::optional<JsonParser>{JsonParser{f}};
+  return std::optional<JsonParser>{JsonParser{content}};
 }
 std::optional<JsonParser> JsonParser::Parser(nlohmann::json obj) noexcept {
   return std::optional<JsonParser>{JsonParser{obj}};
 }
-JsonParser::JsonParser(std::ifstream& f) {
-  this->obj = nlohmann::json::parse(f);
+JsonParser::JsonParser(std::string data) {
```

Comment:
```suggestion
JsonParser::JsonParser(const std::string& data) {
```
albo nawet move (oszczedzaj pamieć i wydajność)

Reply:
+ jak byś chciał sprawdz to w teście

Reply:
![image](https://github.com/user-attachments/assets/403604e3-94c8-493a-8309-7e4dffe43dc0)